IHP/ModelSupport.hs

```diff
@@ -154,6 +155,9 @@ instance InputValue value => InputValue [value] where
 instance InputValue Aeson.Value where
     inputValue json = json |> Aeson.encode |> cs
 
+instance InputValue Scientific where
```

Comment:
Tests for InputValue can be added here https://github.com/digitallyinduced/ihp/blob/master/Test/ModelSupportSpec.hs#L16 :)